ts/store/actions/wallet/transactions.ts

```diff
@@ -19,10 +20,16 @@ export type SelectTransactionForDetails = Readonly<{
   payload: WalletTransaction;
 }>;
 
+export type TransactionError = Readonly<{
```

Comment:
this name is a bit misleading, it seems like an error while performing a transaction, perhaps it's `FETCH_TRANSACTIONS_ERROR` ?